content/broker/broker-transition/_index.nb.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Altinn 3 Formidling overgangsløsning
+linktitle: Overgangsløsning
+description: Altinn 2 Formidling overgang til Altinn 3 Formidling arkitektur. 
+tags: [Formidling, overgang, arkitektur]
+toc: false
+weight: 60
+---
+
+## About
+For eiere av Formidlingstjenester hvis sluttbrukere er avhengige av å bruke Altinn 2, men som ønsker å migrere sin Formidling tjeneste til Altinn 3, tilbyr Altinn en overgangsløsning.
+Dette muliggjør kommunikasjon mellom avsendere og mottakere av filer uavhengig av hvilken Altinn Formidling tjeneste de er integrert i.
+
+Denne løsningen vil tillate tjenesteeiere å koble en eksisterende Altinn 2 Formidlings tjeneste til en Altinn 3 Formidlings ressurs, og for å videresende Altinn 2 Formidlings-forespørsler til Altinn 3 Formidlingstjeneste appen, mens den ikke vesentlig endrer sluttbrukeropplevelsen mot Altinn 2.
+Dette vil da tillate sluttbrukere av Altinn 2 tjenesten å kommunisere med sluttbrukere av Altinn 3 ressursen.
+
+For Altinn 3 brukere vil brukeropplevelsen ikke være annerledes enn om alle bruker Altinn 3.
+
+Altinn 2 brukere vil oppleve noen mindre forskjeller relatert til filoverføring og returnerte verdier sammenlignet med om alle bruker Altinn 2.
```

Comment:
Consider changing "## About" -> "## Om overgangsløsningen"
Consider if "About" should be its own page to mirror the overall Ditaxis structure used for Broker.